CMakeLists.txt

```diff
@@ -30,8 +30,14 @@ hunter_add_package(GTest)
 find_package(GTest CONFIG REQUIRED)
 
 # Add cling support
-find_library (LIBDL_PATH NAMES dl)
-set(CMAKE_CXX_STANDARD_LIBRARIES "${LIBDL_PATH}")
+option(LF_ENABLE_CLING "if set to true, we will link with libdl so that cling works" OFF)
```

Comment:
Should be documented on one of the LehrFEM++ main pages

Reply:
Done here: https://github.com/craffael/lehrfempp/wiki/Building-LehrFEM--